Source/Core/Core/PowerPC/PPCAnalyst.cpp

```diff
@@ -810,10 +801,7 @@ u32 PPCAnalyzer::Analyze(u32 address, CodeBlock *block, CodeBuffer *buffer, u32
 		}
 		else
 		{
-			// ISI exception or other critical memory exception occured (game over)
-			// We can continue on in MMU mode though, so don't spam this error in that case.
-			if (!virtualAddr)
-				ERROR_LOG(DYNA_REC, "Instruction hex was 0!");
+			ERROR_LOG(DYNA_REC, "Instruction hex was 0!");
```

Comment:
Do we need this anymore? We already check for instruction fetch failure, so we should probably just count 0x0 as another example of an invalid instruction?